configs/disease_mesh/2020.07.0.ini

```diff
@@ -0,0 +1,21 @@
+[DEFAULT]
+version=2020.07.0
+
+[preprocess]
+input=data/raw/allMeSH_2020.json
+output=data/processed/disease_mesh.jsonl
+filter_tags=data/processed/mesh_disease_tags.csv
+
+[label_binarizer]
+data=data/processed/disease_mesh.jsonl
+label_binarizer=models/disease_mesh_label_binarizer.pkl
+
+[data]
+train_data_path=data/processed/disease_mesh.jsonl
+test_data_path=
+
+[model]
+label_binarizer_path=models/disease_mesh_label_binarizer.pkl
+model_path=models/disease_mesh_tfidf-svm-2020.07.0
+approach=tfidf-sgd
+parameters={"tfidf__ngram_range": (1,2), "svm__estimator__loss": "log", "svm__estimator__alpha": 1e-9, "tfidf__max_features": 2_000_000}
```

Comment:
why underscore (`2_000_000`) ?

Reply:
why 2M or why I am using the underscore?

Reply:
underscore is ignored by python, it is just for readability

Reply:
ha - didn't know that